images/base/overlay/etc/cont-init.d/10-setup_user.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+gow_log "**** Configure default user ****"
```

Comment:
do we need to `source` the `utils.sh` to be able to use `gow_log` here?

Reply:
Each file in cont-init.d is being sourced in alphabetical order from the `entrypoint.sh` script. Because of this, we only need to have the entrypoint script source the utils.sh file and then each one of these cont-init.d files will have access to it.

Reply:
Ah, that makes sense, thanks!